js/lib/card.js

```diff
@@ -0,0 +1,88 @@
+export function createCard(data) {
+  const cardSection = document.createElement('div')
+  cardSection.classList.add('question-box')
+  cardSection.classList.add('content-box')
```

Comment:
You can use in the method `add()` more strings to add more than one class
Save one line.
```suggestion
  cardSection.classList.add('question-box', 'content-box')
```